src/main/java/com/sforce/cd/apexUnit/client/fileReader/ApexManifestFileReader.java

```diff
@@ -123,11 +123,16 @@ public ApexManifestFileReader(boolean includeTriggers) {
 	 */
 	private void insertIntoTestClassesArray(String strLine, ArrayList<String> testClassList){
 		String tempTestClassId = null;
-		Map<String, String> namespaceAndName = new HashMap<String, String>();
-		namespaceAndName.put("name",strLine);				
+		String[] splitByPeriod = strLine.split("\\.", 2);
+		String namespace = null;
+		String classname = strLine;
+
+		if (splitByPeriod.length == 2) {
```

Comment:
What if the length is not 2? We need to handle all the cases and consider
a. users who dont use namespace
b. users who use a namespace
c. users who use multiple namespaces
d. users who would want to use a namespace as 'default'